docs/sources/features/datasources/loki.md

```diff
@@ -0,0 +1,119 @@
+| _URL_           | The URL of the Loki instance, e.g., `http://localhost:3100`                                                                                   |
+| _Maximum lines_ | Upper limit for number of log lines returned by Loki (default is 1000). Decrease if your browser is sluggish when displaying logs in Explore. |
+
+## Querying Logs
+
+Querying and displaying log data from Loki is available via [Explore](/features/explore).
+Select the Loki data source, and then enter a log query to display your logs.
+
+> Viewing Loki data in dashboard panels is not supported yet, but is being worked on.
+
+### Log Queries
+
+A log query consists of two parts: **log stream selector**, and a **search expression**. For performance reasons you need to start by choosing a log stream by selecting a log label.
+
+The Logs Explorer (the `Log labels` button) next to the query field shows a list of labels of available log streams. An alternative way to write a query is to use the query field's autocomplete - you start by typing a left curly brace `{` and the autocomplete menu will suggest a list of labels. Press the `enter` key to execute the query.
+
+Once the result is returned, the log panel shows a list of log rows and a bar chart where the x-axis shows the time and the y-axis shows the frequency/count.
+
+<div class="medium-6 columns">
+  <video width="800" height="500" controls>
+    <source src="/assets/videos/explore_loki.mp4" type="video/mp4">
+    Your browser does not support the video tag.
+  </video>
+</div>
+
+<br />
+
+### Log Stream Selector
+
+For the label part of the query expression, wrap it in curly braces `{}` and then use the key value syntax for selecting labels. Multiple label expressions are separated by a comma:
+
+`{app="mysql",name="mysql-backup"}`
+
+The following label matching operators are currently supported:
+
+* `=` exactly equal.
+* `!=` not equal.
+* `=~` regex-match.
+* `!~` do not regex-match.
+
+Examples:
+
+* `{name=~"mysql.+"}`
+* `{name!~"mysql.+"}`
+
+The [same rules that apply for Prometheus Label Selectors](https://prometheus.io/docs/prometheus/latest/querying/basics/#instant-vector-selectors) apply for Loki Log Stream Selectors.
+
+Another way to add a label selector, is in the table section, clicking on the **Filter** button beside a label will add the label to the query expression. This even works for multiple queries and will the label selector to each query.
+
+### Search Expression
+
+After writing the Log Stream Selector, you can filter the results further by writing a search expression. The search expression can be just text or a regex expression.
+
+Example queries:
+
+* `{job="mysql"} error`
+* `{name="kafka"} tsdb-ops.*io:2003`
+* `{instance=~"kafka-[23]",name="kafka"} kafka.server:type=ReplicaManager`
+
+## Templating
+
+Template variables are not yet supported by Loki.
+
+## Annotations
+
+Annotations are not yet supported by Loki.
+
+## Configure the Datasource with Provisioning
```

Comment:
Probably a good idea to use the basic auth example from here https://github.com/grafana/loki/blob/master/docs/grafana-provisioning.md#configuring-grafana-via-provisioning

And adding `maxLines` to that.

Reply:
Maybe you can change the loki getting started to point to grafana docs then instead.

Reply:
I would leave the example as is, since k8s seem more likely, and they dont rely on basic auth, AFAIK.